go/chat/utils/utils.go

```diff
@@ -1100,6 +1100,28 @@ func GetDesktopNotificationSnippet(conv *chat1.ConversationLocal, currentUsernam
 	return snippet
 }
 
+func PresentRemoteConversationAsSmallTeamRow(ctx context.Context, rc types.RemoteConversation,
+	username string, useSnippet bool) (res chat1.UIInboxSmallTeamRow) {
+	res.ConvID = rc.GetConvID().String()
+	res.IsTeam = rc.GetTeamType() == chat1.TeamType_SIMPLE
+	res.Name = rc.GetName()
+	res.Name = strings.Replace(res.Name, fmt.Sprintf(",%s", username), "", -1)
```

Comment:
this logic exists in a few places, can you consolidate to just use the helper in utils? 
```
$ git grep '",%s", username'
go/bind/extension.go:258:       item.Name = strings.Replace(item.Name, fmt.Sprintf(",%s", username), "", -1)
go/chat/utils/utils.go:1164:    name = strings.Replace(name, fmt.Sprintf(",%s", username), "", -1)
```